### ISSUE TITLE
[BUG] Checkerboard Texture

### ISSUE THREAD

**Issue by YTG1234** [Juuxel/VanillaParts#18]
Minecraft Version: 1.16.1
Fabric API Version: 0.14.1+build.372-1.16
Mod Version: [1.16.1] Vanilla Parts 1.1.3
Lib Multipart Version: [1.16] v0.4.0
Optifine: No.

I suspect Sodium and/or phosphor.

**Comment by YTG1234** [comment]
Modlist: https://pastebin.com/0Zjt4r6h
@Juuxel

**Comment by qb20nh** [comment]
Works fine with Canvas Renderer

**Comment by Juuxel** [comment]
Yeah, I suspect Sodium might be the cause since LibMultipart uses some Fabric rendering APIs that it might not implement.
